### PR TITLE
points!

### PR DESCRIPTION
**Pull request by greg-schrammel**
Fixes BX-####
Figma link (if any):

## Screen recordings / screenshots

https://github.com/rainbow-me/browser-extension/assets/6232729/d6c01a57-3eee-401a-a1cf-58b1b0e89dcf

### CODE REVIEW

File: src/entries/popup/pages/home/Points/Points.tsx [hunk 1]
```diff
@@ -0,0 +1,401 @@
...
+      </Text>
+      <Text size="10pt" weight="bold" color="labelTertiary">
+        #{position}
+      </Text>
+    </Inline>
+  );
+};
+
+const usePoints = (
+  address: Address = '0x52c717ce5a6b483a890bcdc3114ff140e679b43f',
```
Comment (estebanmino) [inline src/entries/popup/pages/home/Points/Points.tsx:100]: hardcoded address by default?

File: src/entries/popup/pages/home/Points/Points.tsx [hunk 2]
```diff
@@ -0,0 +1,401 @@
...
+              flexDirection="row"
+              alignItems="center"
+              whileTap={{ scale: 0.98 }}
+              whileFocus={{ scale: 1.02 }}
+              whileHover={{ scale: 1.02 }}
+              onTap={() =>
+                copy({
+                  value: data.user.referralCode, // TODO: change to link
+                  title: i18n.t('points.copied_referral_link'),
+                  description: data.user.referralCode, // TODO: change to link
```
Comment (estebanmino) [inline src/entries/popup/pages/home/Points/Points.tsx:236]: 👌

Comment (greg-schrammel) [reply src/entries/popup/pages/home/Points/Points.tsx]: added the link now

**Review by estebanmino — APPROVED** [review]
🪨

**Comment by DanielSinclair** [comment]
With #1170 merged, I think we'll need to use the `featureFlags.points` conditional here. NFT release will go out before Points

**Review by derHowie — APPROVED** [review]
This looks awesome. I think we should make some changes based on watched wallets, though.

If watching I think the 'Your Rank` copy should change to 'Their Rank' and maybe we should hide the referral code components when viewing a watched address.

**Pull request merged**